packages/react/src/preview-card/positioner/usePreviewCardPositioner.ts

```diff
@@ -21,7 +21,6 @@ export function usePreviewCardPositioner(
         }
 
         return mergeReactProps<'div'>(externalProps, {
-          role: 'presentation',
           hidden: !mounted,
```

Comment:
Why was this moved?

Reply:
So it's consistent with other roles - they are placed on popups, not positioners. Is it an issue?

Reply:
For some screen readers, the positioner wrapper needs to have `role="presentation"` to not interfere with the child popup `role` from what I've gathered. It makes it 'inert'/ignored by them. The popup is what should have the ARIA role, but never role=presentation